frontend/src/Components/Upcomingeve/Upcomingevents.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import "./Upcomingeve.css";
+import Card from 'react-bootstrap/Card';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import image1 from "../../images/event1.jpeg";
+import image2 from "../../images/event2.jpeg";
+import image3 from "../../images/event3.jpg";
+
+const Upcomingevents = () => {
+    return(
+        
+            <Container>
+            <Row>
+                <Col>
+                    <Card   id = 'upcomingeve' style={{ width: '18rem' }}  >
```

Comment:
You should create one object for all dummy data, Fetch here and through map/ Create separate component for Cart item.

Reply:
@rutuja2229 make the above changes.